ProcessLib/TimeLoop.cpp

```diff
@@ -648,8 +648,8 @@ void postTimestepForAllProcesses(
         }
         auto& x = *process_solutions[process_id];
         auto& x_dot = *x_dots[process_id];
-        pcs.postTimestep(process_solutions, t, dt, process_id);
         pcs.computeSecondaryVariable(t, dt, x, x_dot, process_id);
+        pcs.postTimestep(process_solutions, t, dt, process_id);
```

Comment:
Despite subtle, this change is very beneficial for the understanding of physical processes. Can this change be understood as the member function `computeSecondaryVariable` serves as named solely for computing secondary variables and `postTimestep` serve for processing local state variables. Is my understanding correct?

Reply:
Yes. I tried to summarize this in the commit message---have a look, can be improved if you suggest something.
The usage of postTimestep is more general, not only pushing states, but the computation of secondary indeed must not be mixed with other things because of the extra initialization call before first time step.

Reply:
IMHO, it is more clear to message like `computation of secondary variables` instead of `computation of some of those`. For me, the stress/strain field is derived from the displacement field and thought of not as a local variable.

Reply:
:+1: Will update with next push.